test/unit/sdk_reporter/test_sdk_reporter.py

```diff
@@ -277,9 +285,5 @@ def test_create_pg_not_invoked_in_non_lambda_case(self):
 
         with self.client_stubber:
             self.subject.refresh_configuration()
-            assert self.subject.is_lambda_one_click_pg_created_during_execution is False
-            try:
-                self.client_stubber.assert_no_pending_responses()
-                assert False
-            except AssertionError:
-                assert True
+            assert self.subject.is_profiling_group_created_during_execution is False
+            assert self.client_stubber.assert_no_pending_responses() is None
```

Comment:
Minor: this warns about no empty line at end of file.

Reply:
sorry, added it!